sdk/python/kfp/_client.py

```diff
@@ -422,6 +422,13 @@ def schedule_pipeline(self, experiment_id, job_name, pipeline_package_path=None,
         params {dict} -- The pipeline parameters (default: {{}})
         pipeline_id {[type]} -- The id of the pipeline which should run on schedule (default: {None})
         namespace {[type]} -- The name space with which the pipeline should run (default: {None})
+        max_concurrency {[type]} -- Max number of concurrent runs scheduled (default: {10})
+        no_catchup {[type]} -- Whether the recurring run should catch up if behind schedule.
+          For example, if the recurring run is paused for a while and re-enabled
+          afterwards. If no_catchup=False, the scheduler will catch up on (backfill) each
+          missed interval. Otherwise, it only schedules the latest interval.
```

Comment:
Just want to make sure I understand it correctly. 'latest interval' actually means 'current interval', right? And the behavior should be if `no_catchup`, then the run won't trigger or backfilled right after this request, instead it will wait for the next trigger time from now.

Reply:
In fact, it will backfill the latest recent interval immediately: https://airflow.apache.org/docs/stable/scheduler.html#backfill-and-catchup